setup.py

```diff
@@ -2,7 +2,7 @@
 
 from tensorcircuit import __version__, __author__
 
-with open("README.md", "r") as fh:
+with open("README.md", "r", encoding = "utf-8") as fh:
```

Comment:
good catch, always want to fix this, always forget

Reply:
:D